### DISCUSSION TITLE
Contributing to add a generic S3-compatible storage provider

### DISCUSSION THREAD

**Discussion by xDefcon** [vitodeploy/vito#293] [Ideas]
Hello,

I'd love to contribute to this project by adding the ability to use any S3-compatible provider such as backblaze or scaleway object storage.

I'd like to know if there's someone working on it. Any feedback/guideline about the development of this feature is appreciated.

Thanks.

**Comment by saeedvaziry** [comment]
I would say let's solve this problem for good.

We already have S3 itself in VitoDeploy. also another S3 compatible storage provider called Wassabi.

Why not creating a new driver (S3-Compatible) which will be used for all these storage providers?

Feel free to drop me a msg on discord

**Comment by rafa-acioly** [comment]
MinIO is also a great tool, it has a active community.

Repository: https://github.com/minio/minio
Official website: https://min.io/
Installation alternatives: https://min.io/open-source/download?platform=kubernetes

**Comment by saeedvaziry** [comment]
PR already merged and released #394
